src/assets/illustrations/feature/process.svg

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg id="a" data-name="BG" xmlns="http://www.w3.org/2000/svg" width="800" height="450" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 800 450">
+  <defs>
+    <clipPath id="b">
+      <rect x="305.23624" y="132.87521" width="186.14801" height="186.14801" fill="none"/>
+    </clipPath>
+    <clipPath id="c">
+      <rect x="305.23624" y="132.87521" width="186.14801" height="186.14801" fill="none"/>
+    </clipPath>
+  </defs>
```

Comment:
This design shouldn't require any clip paths. You may need to clean up your Illustrator file, if you have clipping masks or anything like that you can probably simplify.